PowerEditor/src/ScintillaComponent/ScintillaEditView.h

```diff
@@ -681,16 +681,12 @@ friend class Finder;
 	};
 
 	void setMakefileLexer() {
-		execute(SCI_SETLEXER, SCLEX_MAKEFILE);
-		makeStyle(L_MAKEFILE);
+		setLexer(SCLEX_MAKEFILE, L_MAKEFILE, LIST_NONE);

```

Comment:
needs review, if this could be done, "needed" because of compilation issues if code is placed within header file